### ISSUE TITLE
problem with updating DB after upgrade from v4.12.1 => v4.13.9 

### ISSUE THREAD

**Issue by prathers** [globaleaks/GlobaLeaks#3631]
### What version of GlobaLeaks are you using?

4.13.9

### What operating system(s) are you seeing the problem on?

Linux

### Describe the issue

After running sudo apt-get update && sudo apt-get install globaleaks sudo globaleaks start failed. Did the reinstall as recommended and after changing permissions, I get this in globaleaks.log:
```
2023-09-15 08:39:24+0000 [-] twistd 22.1.0 (/usr/bin/python3 3.10.12) starting up.
2023-09-15 08:39:24+0000 [-] reactor class: twisted.internet.epollreactor.EPollReactor.
2023-09-15 08:39:26+0000 [-] [E] Found an already initialized database version: 65
2023-09-15 08:39:26+0000 [-] [E] Performing schema migration from version 65 to version 66
2023-09-15 08:39:26+0000 [-] [I] Updating DB from version 65 to version 66
2023-09-15 08:39:27+0000 [-] [E] Failure: (sqlite3.OperationalError) no such column: whistleblowerfile.filename\n[SQL: SELECT count(*) AS count_1 \nFROM (SELECT whistleblowerfile.id AS whistleblowerfile_id, whistleblowerfile.internaltip_id AS whistleblowerfile_internaltip_id, whistleblowerfile.name AS whistleblowerfile_name, whistleblowerfile.filename AS whistleblowerfile_filename, whistleblowerfile.size AS whistleblowerfile_size, whistleblowerfile.content_type AS whistleblowerfile_content_type, whistleblowerfile.creation_date AS whistleblowerfile_creation_date, whistleblowerfile.access_date AS whistleblowerfile_access_date, whistleblowerfile.description AS whistleblowerfile_description, whistleblowerfile.visibility AS whistleblowerfile_visibility, whistleblowerfile.new AS whistleblowerfile_new \nFROM whistleblowerfile) AS anon_1]\n(Background on this error at: https://sqlalche.me/e/14/e3q8)
2023-09-15 08:39:27+0000 [-] [E] Verbose exception traceback:
2023-09-15 08:39:27+0000 [-] [I] Traceback (most recent call last):\n\n  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/base.py", line 1802, in _execute_context\n    self.dialect.do_execute(\n\n  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/default.py", line 732, in do_execute\n    cursor.execute(statement, parameters)\n\nsqlite3.OperationalError: no such column: whistleblowerfile.filename\n\n\nThe above exception was the direct cause of the following exception:\n\n\nTraceback (most recent call last):\n\n  File "/usr/lib/python3/dist-packages/globaleaks/db/__init__.py", line 94, in update_db\n    migration.perform_migration(db_version)\n\n  File "/usr/lib/python3/dist-packages/globaleaks/db/migration.py", line 224, in perform_migration\n    migration_script = MigrationModule.MigrationScript(migration_mapping, version, session_old, session_new)\n\n  File "/usr/lib/python3/dist-packages/globaleaks/db/migrations/update.py", line 40, in __init__\n    self.entries_count[model_name] = self.session_old.query(self.model_from[model_name]).count()\n\n  File "/usr/lib/python3/dist-packages/sqlalchemy/orm/query.py", line 3145, in count\n    return self._from_self(col).enable_eagerloads(False).scalar()\n\n  File "/usr/lib/python3/dist-packages/sqlalchemy/orm/query.py", line 2879, in scalar\n    ret = self.one()\n\n  File "/usr/lib/python3/dist-packages/sqlalchemy/orm/query.py", line 2856, in one\n    return self._iter().one()\n\n  File "/usr/lib/python3/dist-packages/sqlalchemy/orm/query.py", line 2894, in _iter\n    result = self.session.execute(\n\n  File "/usr/lib/python3/dist-packages/sqlalchemy/orm/session.py", line 1692, in execute\n    result = conn._execute_20(statement, params or {}, execution_options)\n\n  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/base.py", line 1614, in _execute_20\n    return meth(self, args_10style, kwargs_10style, execution_options)\n\n  File "/usr/lib/python3/dist-packages/sqlalchemy/sql/elements.py", line 325, in _execute_on_connection\n    return connection._execute_clauseelement(\n\n  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/base.py", line 1481, in _execute_clauseelement\n    ret = self._execute_context(\n\n  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/base.py", line 1845, in _execute_context\n    self._handle_dbapi_exception(\n\n  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/base.py", line 2026, in _handle_dbapi_exception\n    util.raise_(\n\n  File "/usr/lib/python3/dist-packages/sqlalchemy/util/compat.py", line 207, in raise_\n    raise exception\n\n  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/base.py", line 1802, in _execute_context\n    self.dialect.do_execute(\n\n  File "/usr/lib/python3/dist-packages/sqlalchemy/engine/default.py", line 732, in do_execute\n    cursor.execute(statement, parameters)\n\nsqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such column: whistleblowerfile.filename\n[SQL: SELECT count(*) AS count_1 \nFROM (SELECT whistleblowerfile.id AS whistleblowerfile_id, whistleblowerfile.internaltip_id AS whistleblowerfile_internaltip_id, whistleblowerfile.name AS whistleblowerfile_name, whistleblowerfile.filename AS whistleblowerfile_filename, whistleblowerfile.size AS whistleblowerfile_size, whistleblowerfile.content_type AS whistleblowerfile_content_type, whistleblowerfile.creation_date AS whistleblowerfile_creation_date, whistleblowerfile.access_date AS whistleblowerfile_access_date, whistleblowerfile.description AS whistleblowerfile_description, whistleblowerfile.visibility AS whistleblowerfile_visibility, whistleblowerfile.new AS whistleblowerfile_new \nFROM whistleblowerfile) AS anon_1]\n(Background on this error at: https://sqlalche.me/e/14/e3q8)\n
2023-09-15 08:39:27+0000 [-] Server Shut Down.
```

**Comment by evilaliv3** [comment]
Thank you.

This compatibility issue is going to fixed with release 4.13.10 that is going to be released by today.

**Comment by prathers** [comment]
Confirmed. The v4.13.10 solved the problem
